Templates/AppSource App/.github/workflows/CreateRelease.yaml

```diff
@@ -304,11 +308,13 @@ jobs:
       - name: Create Release Branch
         run: |
           $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
-          git checkout -b ${{ needs.CreateRelease.outputs.releaseBranch }}
+          $releaseBranch = '${{ github.event.inputs.releaseBranchPrefix }}${{ needs.CreateRelease.outputs.releaseVersion }}'
```

Comment:
Isn't there a security risk when we use inputs inline in code?
I think this should be converted to env and then used as env

Reply:
Not sure what the risk is here. The workflow has only the `workfow_dispatch` trigger.

Anyway, I extracted an env variable.